### DISCUSSION TITLE
Installing problem

### DISCUSSION THREAD

**Discussion by woop120** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#422] [Q&A]
![image](https://user-images.githubusercontent.com/95972277/145681467-2164c50c-235f-4960-85f6-f1cb2ee3f74b.png)

Why do i get this error pop-up when trying to  enable the add-on? I just succesfully installed it and just need to check the enable box but it wont let me. Blender version is 3.0.0

**Comment by bsavery** [comment]
Use the weekly build with blender 3.0.

**Reply by woop120** [reply]
Thanks but where can i find that? I've been looking i can't find a "weekly build"

**Reply by bsavery** [reply]
https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/releases

**Reply by woop120** [reply]
Awesome, thanks man
